scipy/stats/tests/test_morestats.py

```diff
@@ -2592,30 +2588,39 @@ def test_directionalmean_correctness(self):
                          np.sin(incl)),
                         axis=1)
 
-        directional_mean = stats.directionalmean(data)
-        mean_rounded = np.round(directional_mean, 3)
+        dirstats = stats.directional_stats(data)
+        directional_mean = dirstats.mean_direction
+        mean_rounded = np.round(directional_mean, 4)
 
-        reference_mean = np.array([0.298, -0.135, -0.945])
+        reference_mean = np.array([0.2984, -0.1346, -0.9449])
         assert_allclose(mean_rounded, reference_mean)
 
-    def test_directionalmean_2d(self):
-        # Test that for circular data directionalmean yields
-        # same result as circmean
+        expected_var = 0.025335389565304012
```

Comment:
I too can't find it anywhere in the paper. @dschmitz89 Do you remember? I rebased on top of your commits which included this.

Reply:
I think I computed that with `circvar`. The reference does not contai  it indeed.

Reply:
How did you compute the circular variance using 3-D vector angles using `circvar`? (I thought `circvar` only considers points on a 2-D plane i.e. takes only one angle)

Reply:
You are completely correct, I was misreading this conversation before. Updated my answer above.

Reply:
I'd remove it, then. We're already checking the mean direction, which shares almost all of the same implementation, and just to be safe, we check that the function against circvar in 2D.  If we want more, the options I see for testing in n-D are 1) checking against a very short and simple reference calculation, written in the test itself, that closely matches the formulas from the reference (not very useful here, since the function itself already does that) and 2) checking properties, like that it gives 1 when all vectors are aligned and 0 when they're evenly spread.

Reply:
Ok, lets do so. Sorry if it the test seemed misleading so far.

Reply:
No worries @dschmitz89! I have added some tests to check special cases like 0 and 1 variance and also a test that checks the directional variance against `circvar`. Let me know if it looks good @mdhaber @dschmitz89 .